source/D2Game/src/GAME/Clients.cpp

```diff
@@ -677,19 +677,19 @@ void __fastcall CLIENTS_RemoveClientFromGame(D2GameStrc* pGame, int32_t nClientI
             if (D2_VERIFY(gpD2EventCallbackTable_6FD45830->pfLeaveGame))
             {
                 gpD2EventCallbackTable_6FD45830->pfLeaveGame(
-                    &pClientToRemove->pClientInfo,
-                    pGame->nServerToken,
-                    pPlayer->dwClassId, nPlayerLevel, nPlayerExperience, HIDWORD(nPlayerExperience),
-                    pClientToRemove->nSaveFlags, pClientToRemove->szName,
-                    (const char*)&pClientToRemove->tCharacterInfo, pClientToRemove->bUnlockCharacter,
-                    0,
-                    0,
-                    pClientToRemove->szAccount, pClientToRemove->unk0x60, &pClientToRemove->unk0x190);
+						&pClientToRemove->pClientInfo,
+						pGame->nGameId,
+						pPlayer->dwClassId, nPlayerLevel, nPlayerExperience, HIDWORD(nPlayerExperience),
+						pClientToRemove->nSaveFlags, pClientToRemove->szName,
+						(const char*)&pClientToRemove->tCharacterInfo, pClientToRemove->bUnlockCharacter,
+						0,
+						0,
+						pClientToRemove->szAccount, pClientToRemove->nDatabaseCharacterId, &pClientToRemove->nPlayerMark);
```

Comment:
These are grim, but I'm also lost as to whether the project uses tabs or spaces.
Don't see a way to change continuation indent to spaces in my editor.